django_emarsys/checks.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from django.apps import apps
+from django.conf import settings
+from django.core.checks import register, Critical, Debug, Error, Info, Warning
+
+log = logging.getLogger(__file__)
+
+
+@register()
+def check_settings(app_configs):
+    return validate_settings()
+
+
+def add_message(messages, message):
+    messages.append(message)
+    log_message = {Debug: log.debug,
+                   Info: log.info,
+                   Warning: log.warning,
+                   Error: log.error,
+                   Critical: log.critical}[type(message)]
+
+    log_message(message.msg)
```

Comment:
Hmm, I think if it made sense to log failing checks, Django would already do that. What's the rationale? I remember you mentioned that, but I don't remember the rationale.

I'd rather not log. Django isn't going to start up with failing checks anyway.